Jenkinsfile

```diff
@@ -132,7 +126,7 @@ pipeline {
           
           //warnings plugin
           rtMaven.run pom: 'pom.xml', goals: '--batch-mode -V -e checkstyle:checkstyle pmd:pmd pmd:cpd findbugs:findbugs com.github.spotbugs:spotbugs-maven-plugin:3.1.7:spotbugs -Dmaven.repo.local=.m2'
-
+          
```

Comment:
normally, we try to avoid unnecessary white space like here…

Reply:
can't explain where they come from. thanks for noticing. Will catch up on this...